Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -35,6 +35,13 @@ private struct RemoteTabsPanelUX {
 
     // Temporary placeholder for strings removed in Bug 1193456.
     private let CreateAccountString = NSLocalizedString("Create an account", comment: "See http://mzl.la/1Qtkf0j")
+
+    // Backup and active strings added in Bug 1205294.
+    static let EmptyStateInstructionsSyncTabsPasswordsBookmarksString = NSLocalizedString("Sync your tabs, bookmarks, passwords and more.", comment: "See http://mzl.la/1Qtkf0j")
```

Comment:
Not sure if we want to use a link in the comment to describe the context of the string or is this temporary?